traveler-ios-ui/TravelerKitUI/ViewControllers/BookingOptionsViewController.swift

```diff
@@ -28,6 +28,7 @@ class BookingOptionsViewController: UIViewController {
     }
     var optionError: Error?
     var selectedOption: BookingOption?
+    var selectedIndexPath: IndexPath?
```

Comment:
private

Reply:
I think you can do without this var as well..just use the selectedOption?

Either hold on to the selected index path or the selected model..not both